app/src/organisms/Desktop/Camera/CameraControls/PreviewSettings/hooks/usePreviewImage.ts

```diff
@@ -1,18 +1,33 @@
 import { useState } from 'react'
 
+import { useCapturePreviewImage } from '@opentrons/react-api-client'
+
+import type { CameraImageSettings } from '@opentrons/api-client'
+
 export interface UsePreviewImageResult {
   imgPath: string | undefined
   isLoading: boolean
   takePhoto: () => void
 }
 
-export function usePreviewImage(): UsePreviewImageResult {
-  const [isLoading, setIsLoading] = useState(false)
+export function usePreviewImage(
+  settings: CameraImageSettings
+): UsePreviewImageResult {
   const [imgPath, setImgPath] = useState<string | undefined>(undefined)
+  const [isLoading, setIsLoading] = useState(false)
 
-  const takePhoto = (): void => {
-    setIsLoading(true)
+  const { mutate: captureImage } = useCapturePreviewImage({
+    onSuccess: response => {
+      setIsLoading(false)
+    },
```

Comment:
```suggestion
onSettled: () => {
setIsLoading(false)
}
```
slightly cleaner, since we can nix this from `onError` too and only have to maintain one call site.